config/initializers/devise.rb

```diff
@@ -34,7 +34,7 @@
   # session. If you need permissions, you should implement that in a before filter.
   # You can also supply a hash where the value is a boolean determining whether
   # or not authentication should be aborted when the value is not present.
-  # config.authentication_keys = [:email]
+  config.authentication_keys = [:logins]
```

Comment:
I think that's a typo.  At Costco right now, I will fix it when I come
back.  I was wondering why it broke.

On Sep 13, 2016 6:27 PM, "Brian Kung" <notifications@github.com> wrote:

> In config/initializers/devise.rb
> <https://github.com/briankung/projectcredo/pull/76#discussion_r78663384>:
>
> > @@ -34,7 +34,7 @@
> >    # session. If you need permissions, you should implement that in a before filter.
> >    # You can also supply a hash where the value is a boolean determining whether
> >    # or not authentication should be aborted when the value is not present.
> > -  # config.authentication_keys = [:email]
> > +  config.authentication_keys = [:logins]
>
> Logins
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/76/files/f1e021a4a686f1c3e2ed35ec58fa2814489047ac#r78663384>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZFphDpxAspzuvgbOr8f7JKE9ot8Iks5qpzFIgaJpZM4J7Lyl>
> .
>